### PR TITLE
[MIG] stock_owner_restriction: Migration to 16.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
@qrtl

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by yostashiro — APPROVED** [review]
Functional test in a local environment. 👍

**Review by Yadier-Tecnativa — APPROVED** [review]
Approve

File: stock_owner_restriction/models/stock_picking_type.py [hunk 1]
```diff
@@ -9,9 +9,9 @@ class StockPickingType(models.Model):
 
     owner_restriction = fields.Selection(
         [
-            ("standard_behavior", "Standard behavior"),
-            ("unassigned_owner", "Unassigned owner"),
-            ("picking_partner", "Picking partner"),
+            ("standard_behavior", "Standard Behavior"),
```
Comment (pedrobaeza) [inline stock_owner_restriction/models/stock_picking_type.py]: Please don't change this that forces to retranslate, and it's not more correct putting both in upper.

**Comment by pedrobaeza** [comment]
Please don't create new commits. Just amend the last one. Now you need to squash and a new CI build will be launched, wasting CPU time.

**Review by sergio-teruel — APPROVED** [review]
Functional review ok

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Pull request merged**